src/node.cc

```diff
@@ -1383,27 +1384,28 @@ void AppendExceptionLine(Environment* env,
                      linenum,
                      sourceline_string);
   CHECK_GE(off, 0);
```

Comment:
This can still cause an assertion when `sourceline_string` is really long and tries to write past `sizeof(arrow)`.